pkg/controller/siddhiprocess/artifacts.go

```diff
@@ -153,7 +152,7 @@ func (rsp *ReconcileSiddhiProcess) UpdateIngress(
 
 	var ingressPaths []extensionsv1beta1.HTTPIngressPath
 	for _, port := range siddhiApp.ContainerPorts {
-		path := "/" + strings.ToLower(siddhiApp.Name) + "/" + strconv.Itoa(int(port.ContainerPort)) + "/"
+		path := "/" + strings.ToLower(siddhiApp.Name) + "/" + strconv.Itoa(int(port.ContainerPort)) + "(/|$)(.*)"
```

Comment:
Can you elaborate on how this path would be used in re-write target

Reply:
When the request URI comes to the ingress, the substrings within the path that must be passed to the rewritten path maps using captured groups like $1, $2, ... , $n. When we use `(/|$)(.*)`, anything within `(.*)` maps as $2. This $2 I passed to the rewrite-target annotation. 

That is what happens inside. For more reference use following links.

https://kubernetes.github.io/ingress-nginx/examples/rewrite/
https://kubernetes.github.io/ingress-nginx/user-guide/ingress-path-matching/
https://www.regular-expressions.info/refcapture.html